Client/src/main/java/org/projectcardboard/client/models/card/TempDeck.java

```diff
@@ -30,11 +29,7 @@ public String getHeroId() {
         return heroId;
     }
 
-    public String getItemId() {
-        return itemId;
-    }
-
-    public List<String> getOthersIds() {
```

Comment:
Did you double check we arent using this method anywhere? sometimes IntelliJ reports that a method isnt being use when in fact it is (e.g. the setNewNextCard method on the client side)

Reply:
getItemId? No instances in the project.